.github/workflows/codeql-analysis.yml

```diff
@@ -2,41 +2,44 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [master]
+    branches: [ main ]
   pull_request:
     # The branches below must be a subset of the branches above
-    branches: [master]
+    branches: [ main ]
   schedule:
-    - cron: "30 0 * * 0"
+    - cron: '30 0 * * 0'
+    
+permissions:
+  contents: read
 
 jobs:
   analyze:
+    permissions:
+      security-events: write  # for github/codeql-action/autobuild to send a status report
     name: Analyze
     runs-on: ubuntu-latest
-    permissions:
-      actions: read
-      contents: read
-      security-events: write
 
     strategy:
       fail-fast: false
       matrix:
-        language: ["go"]
+        language: [ 'go' ]
```

Comment:
This would totally depend on the individual repos, i.e. some repos also have `javascript`. Hence in future, if we have support for `javascript`, we will extend the support to the additional languages. 

For consistency across all the repos, we have used the same format of using matrix.languages instead of hardcoding them.

Reply:
Aha, wasn't aware about the consistency requirement. 👍 
Makes sense